VoxelSniperCore/src/main/java/com/github/kevindagame/util/Utils.kt

```diff
@@ -3,14 +3,21 @@ package com.github.kevindagame.util
 import java.util.*
 
 object Utils {
+    @JvmStatic
     fun <T> unmodifiableList(): List<T> {
         return Collections.unmodifiableList(ArrayList<T>())
```

Comment:
I don't really like doing it like this, since in kotlin listOf() is already immutable. Unless explicitly specified, any list is immutable

Reply:
Did not know that, will try to use that instead.

Reply:
We still need it in some places in java files I think, but I changed it wherever I can https://github.com/KevinDaGame/VoxelSniper-Reimagined/pull/197/commits/6310a268b037feb1254f6e2d96297b95952e1a18